test/testcafe/framework/page-objects/IdentityPageObject.js

```diff
@@ -207,12 +203,23 @@ export default class IdentityPageObject extends BasePageObject {
     await this.t.click(Selector(ENROLL_SELECTOR));
   }
 
+  getUnlockAccountLink() {
+    if(userVariables.v3) {
+      return this.form.getLink('Unlock account?');
+    }
+    return Selector(UNLOCK_ACCOUNT);
```

Comment:
The getLink with the text doesn't work for v2?

Reply:
this is for a custom link so the link text its looking for is different so I will refactor this